pact/src/main/java/bio/terra/profile/consumer/SamStatusApiPactTest.java

```diff
@@ -0,0 +1,64 @@
+package bio.terra.profile.consumer;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.PactTestExecutionContext;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit.ConsumerPactTest;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.broadinstitute.dsde.workbench.client.sam.api.StatusApi;
+import org.junit.Assert;
+
+import java.io.IOException;
+
+
+/**
+ * The only way I've been able to get this to generate a pact successfully so far,
+ * has been running it directly in my IDE
+ * If I put it in a test folder, it doesn't get detected as containing any tests
+ * In this folder, the issue is that it isn't set up with a 'main' method
+ */
+public class SamStatusApiPactTest extends ConsumerPactTest {
+
+
+  @Pact(consumer = "bpm-consumer")
```

Comment:
I can also delete the consumer annotation and it still generates the pact.

Reply:
that worked for me, too
I wonder why some examples have both (I just double checked to make sure I wasn't miss-remembering)

Reply:
I don't know… maybe we will figure out more as we get into more complicated usage. Have the annotation just seemed to duplicate the methods (though everything I say should be taken with a huge grain of salt because I haven't read about Pact yet!).

Reply:
Figured it out:
The 'official' example has the annotation on the alternate usage. 
The example that had both was an unofficial example (although it was linked from the official page).
So the example was confused.